job_definitions/index_services.yml

```diff
@@ -9,8 +9,8 @@
     parameters:
       - string:
           name: INDEX
-          default: g-cloud
-          description: "Index name to use (eg 'g-cloud-2017-05-22')"
+          default: {{ default_index_name }}
```

Comment:
Are we planning to switch the search-api to look in `g-cloud-9` instead of `g-cloud`?